features/admin/deactivate_supplier_user.feature

```diff
@@ -0,0 +1,26 @@
+@deactivate-supplier-user
+Feature: Deactivate a supplier's contributor
+
+Scenario Outline: Correct users can deactivate a supplier's contributor
+  Given I am logged in as the production <role> user
+  # TODO get here from /admin
```

Comment:
Would be nice to fill the TODO steps to click through from the home page now admin links are finalised.

Reply:
Done